src/plugins/intel_gpu/src/graph/fully_connected.cpp

```diff
@@ -104,18 +104,20 @@ layout fully_connected_inst::calc_output_layout(fully_connected_node const& node
     }
 
     if (input_layout.is_dynamic()) {
-        auto rank = input_layout.get_rank();
+        auto rank = input_layout.get_partial_shape().size();
```

Comment:
I am curious why you are editing this calc_output_layout API instead of the other one returning vector<layout> ? 
In the future the API to be used by dynamic shape will be the one returning vector<layout>not this one.

Reply:
We didn't have new shape infer for FC at the moment of implementation, and eltwise shape infer is not merged yet.

Reply:
I see, Now we have https://github.com/openvinotoolkit/openvino/pull/12772/files this. Eltwise is WIP.